packages/editor/src/components/post-sticky/index.js

```diff
@@ -29,7 +29,7 @@ export default function PostSticky() {
 			<CheckboxControl
 				className="editor-post-sticky__checkbox-control"
 				label={ __( 'Sticky' ) }
-				help={ __( 'Pin this post to the top of the blog' ) }
+				help={ __( 'Pin this post to the top of the blog.' ) }
```

Comment:
There's a proposal to change the help text; sharing it in case you're interested, @afercia - #69300.